tests/Admin/Extension/Gedmo/DeprecatedTranslatableAdminExtensionTest.php

```diff
@@ -0,0 +1,131 @@
+        $this->expectDeprecation('Not passing an instance of "Gedmo\Translatable\TranslatableListener" as second argument to "Sonata\TranslationBundle\Admin\Extension\Gedmo\TranslatableAdminExtension::__construct()" is deprecated since sonata-project/translation-bundle 2.x and will be mandatory in 3.0.');
+        $this->expectDeprecation('Not passing an instance of "Doctrine\Persistence\ManagerRegistry" as third argument to "Sonata\TranslationBundle\Admin\Extension\Gedmo\TranslatableAdminExtension::__construct()" is deprecated since sonata-project/translation-bundle 2.x and will be mandatory in 3.0.');
+
+        $this->extension = new TranslatableAdminExtension(
+            $translatableChecker
+        );
+
+        $request = new Request();
+        $request->query->set(AbstractTranslatableAdminExtension::TRANSLATABLE_LOCALE_PARAMETER, 'es');
+
+        $this->admin = $this->getMockBuilder(AdminInterface::class)
+            ->addMethods(['getConfigurationPool'])
+            ->getMockForAbstractClass();
+        $this->admin->method('getRequest')->willReturn($request);
+        $this->admin->method('hasRequest')->willReturn(true);
+
+        $this->translatableListener = new TranslatableListener();
+
+        $container = new Container();
+        $container->set('stof_doctrine_extensions.listener.translatable', $this->translatableListener);
+
+        $objectManager = $this->createStub(ObjectManager::class);
+        $managerRegistry = $this->createStub(ManagerRegistry::class);
+        $managerRegistry
+            ->method('getManagerForClass')
+            ->willReturn($objectManager);
+
+        $container->set('doctrine', $managerRegistry);
+
+        $pool = $this->createStub(Pool::class);
+        $pool->method('getContainer')->willReturn($container);
+
+        $this->admin
+            ->method('getConfigurationPool')
+            ->willReturn($pool);
+
+        $this->object = new ModelTranslatable();
+    }
+
+    public function testSetLocaleForTranslatableObject(): void
+    {
+        $this->extension->alterNewInstance($this->admin, $this->object);
+
+        $this->assertSame('es', $this->object->getLocale());
+    }
+
+    public function testAlterObjectForTranslatableObject(): void
+    {
+        $this->extension->alterObject($this->admin, $this->object);
+
+        $this->assertSame('es', $this->object->getLocale());
+    }
+
+    public function testConfigureQuery(): void
+    {
+        $query = $this->createStub(ProxyQueryInterface::class);
+
+        $this->extension->configureQuery($this->admin, $query);
+
+        self::assertSame('es', $this->translatableListener->getListenerLocale());
+        self::assertFalse($this->translatableListener->getTranslationFallback());
```

Comment:
We should be consistent about the way we call the assertrions.
```suggestion
        $this->assertSame('es', $this->translatableListener->getListenerLocale());
        $this->assertFalse($this->translatableListener->getTranslationFallback());
```